### PR TITLE
Add and split Solr search information

### PR DESCRIPTION
**Pull request by Firesphere**

### CODE REVIEW

**Comment by Firesphere** [comment]
@wernerkrauss Could you please at least acknowledge the PR? ;)

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Contributions welcome, please send a pull request.
 * [Stack Overflow](https://stackoverflow.com/questions/tagged/silverstripe)
 * [Silverstripe User Slack](https://silverstripe-users.slack.com/)
   * [Invitation to Silverstripe User Slack](https://www.silverstripe.org/community/slack-signup)
-  * [Slack Archive](https://slackarchive.silverstripe.org)
+  * ~[Slack Archive](https://slackarchive.silverstripe.org)~ Not active anymore due to Slack API changes
```
Comment (wernerkrauss) [inline README.md]: Although it's very informative that the archive doesn't exist anymore, awesome guidelines don't allow dead links. ATM I get a 520 error on cloudflare when I try to access the slack archive

**Comment by wernerkrauss** [comment]
Could you please rebase on my latest changes to the list?

**Comment by Firesphere** [comment]
Applied all feedback and linting fixes.

**Comment by Firesphere** [comment]
p.s.

I renamed the incorrectly spelled "LICENCE" file to "LICENSE.md" ;)

**Comment by Firesphere** [comment]
p.p.s.

@wernerkrauss , I would appreciate it if you tagged valid PR's with `hacktoberfest-accepted` ;)

File: README.md [hunk 2]
```diff
@@ -92,11 +92,24 @@ Outdated items can be found in the [archive](ARCHIVE.md).
...
+#### Newer Solr versions
+- [Solr search](https://github.com/firesphere/silverstripe-solr-search) - Full fledged search interface for Solr search, supporting up to the latest Solr version.
+  - Documentation
+    - [API Documentation](https://firesphere.github.io/solr-api/)
+    - [Usage Documentation](https://firesphere.github.io/solr-docs/)
+  - Submodules
+    - [Fulltext Search compatibility](https://github.com/Firesphere/silverstripe-solr-compatibility) - Make transitioning from the original Fulltext Search module easier.
+    - [Subsites](https://github.com/Firesphere/silverstripe-subsite-solr) - Adds support for subsites.
+    - [Fluent](https://github.com/Firesphere/silverstripe-fluent-solr) - Adds support for Fluent.
+    - [Fine grained member permissions](https://github.com/Firesphere/silverstripe-solr-member-permissions) - Adds per-member based permission instead of Group based permissions.
```
Comment (wernerkrauss) [inline README.md]: Group should be lowercase, like member in a normal sentence

File: README.md [hunk 3]
```diff
@@ -92,11 +92,24 @@ Outdated items can be found in the [archive](ARCHIVE.md).
...
+
+#### Newer Solr versions
+- [Solr search](https://github.com/firesphere/silverstripe-solr-search) - Full fledged search interface for Solr search, supporting up to the latest Solr version.
+  - Documentation
+    - [API Documentation](https://firesphere.github.io/solr-api/)
+    - [Usage Documentation](https://firesphere.github.io/solr-docs/)
+  - Submodules
+    - [Fulltext Search compatibility](https://github.com/Firesphere/silverstripe-solr-compatibility) - Make transitioning from the original Fulltext Search module easier.
+    - [Subsites](https://github.com/Firesphere/silverstripe-subsite-solr) - Adds support for subsites.
+    - [Fluent](https://github.com/Firesphere/silverstripe-fluent-solr) - Adds support for Fluent.
```
Comment (wernerkrauss) [inline README.md]: Why is "Fluent" written uppercase?

File: README.md [hunk 4]
```diff
@@ -92,11 +92,24 @@ Outdated items can be found in the [archive](ARCHIVE.md).
...
+- [Silverstripe Searchable](https://github.com/i-lateral/silverstripe-searchable) - Adds more complex site searching using the Silverstripe ORM. With dedicated templates for search results across multiple search objects.
+- [Searchable DataObjects](https://github.com/g4b0/silverstripe-searchable-dataobjects) - Fast and simple MySQL based search. Useful for single language sites.
+
+#### Solr 4
+- [Fulltext Search](https://github.com/silverstripe/silverstripe-fulltextsearch) - Full fledged search interface for Solr4.
+- [Fulltext Search Local Solr](https://addons.silverstripe.org/add-ons/silverstripe/fulltextsearch-localsolr) - Easy to install Solr4 instance for local development.
+
+#### Newer Solr versions
+- [Solr search](https://github.com/firesphere/silverstripe-solr-search) - Full fledged search interface for Solr search, supporting up to the latest Solr version.
+  - Documentation
```
Comment (wernerkrauss) [inline README.md]: I wonder if we need the whole documentation and submodules. Are they awesome enough for this list? Or should they be linked from the module? 

I mean, we assume an awesome item to have proper documentation, regarding to the "awesome list specs". 

Instead you could write: 
> Full fledged search interface for Solr search, supporting up to the latest Solr version. With submodules for e.g. subsites, fluent and fulltext search compatibility.

File: README.md [hunk 5]
```diff
@@ -92,11 +92,24 @@ Outdated items can be found in the [archive](ARCHIVE.md).
...
-### Site Search
-- [Fulltext Search](https://github.com/silverstripe/silverstripe-fulltextsearch) -  Full fledged search interface for search backends like SOLR or elastic search.
-- [Fulltext Search Local SOLR](https://addons.silverstripe.org/add-ons/silverstripe/fulltextsearch-localsolr) -  Easy to install SOLR instance for local development.
-- [Silverstripe Searchable](https://github.com/i-lateral/silverstripe-searchable) -  Adds more complex site searching using the Silverstripe ORM. With dedicated templates for search results across multiple search objects.
-- [Searchable DataObjects](https://github.com/g4b0/silverstripe-searchable-dataobjects) -  Fast and simple MySQL based search. Useful for single language sites.
+### Site Search 
+- [Silverstripe Searchable](https://github.com/i-lateral/silverstripe-searchable) - Adds more complex site searching using the Silverstripe ORM. With dedicated templates for search results across multiple search objects.
+- [Searchable DataObjects](https://github.com/g4b0/silverstripe-searchable-dataobjects) - Fast and simple MySQL based search. Useful for single language sites.
+
+#### Solr 4
```
Comment (wernerkrauss) [inline README.md:99]: Did you update the Content section? See (updated) [CONTRIBUTING.md](https://github.com/wernerkrauss/awesome-silverstripe/blob/main/CONTRIBUTING.md)

> npx doctoc --no-title --maxlevel 3 README.md

**Comment by wernerkrauss** [comment]
> I renamed the incorrectly spelled "LICENCE" file to "LICENSE.md" ;)
This is, according to (Leo)[https://dict.leo.org/englisch-deutsch/Lizenz] an American vs. British English issue. No problem with renaming, it was correct before.

>  I would appreciate it if you tagged valid PR's with hacktoberfest-accepted ;)
Of course, but I already tagged this repo "hacktoberfest", isn't this enough for your PR to be counted?

**Comment by Firesphere** [comment]
> Of course, but I already tagged this repo "hacktoberfest", isn't this enough for your PR to be counted?

Sadly, no, either tag as accepted, or merge it

>  This is, according to (Leo)[https://dict.leo.org/englisch-deutsch/Lizenz] an American vs. British English issue. No problem with renaming, it was correct before.

According to Awesome:
> Place a file named license or LICENSE in the repo root with the license text.

**Review by wernerkrauss — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Firesphere) [reply README.md]: That is fixed in the rebase

File: README.md (hunk 2 above)
Comment (Firesphere) [reply README.md]: In theory, true. In practice, it refers to a class and in that regards `Member` should actually be upper case.

It's debatable, but I'm happy to make it all lower case :)

File: README.md (hunk 3 above)
Comment (Firesphere) [reply README.md]: Because it's an external repository. It's like a name, but happy to lower case it.

File: README.md (hunk 4 above)
Comment (Firesphere) [reply README.md]: Fair call, I'm not sure either way. I would personally at least leave the documentation in, but remove the submodules?

File: README.md (hunk 5 above)
Comment (Firesphere) [reply README.md:99]: Yes, it uses the git hook, so if it's not, that's odd. I can easily re-do it manually :)

File: README.md (hunk 2 above)
Comment (Firesphere) [reply README.md]: I'm just gonna take all the sub points out. I think that's the better way to go :)

File: README.md (hunk 5 above)
Comment (wernerkrauss) [reply README.md:99]: my bad, I guess it's one level too deep and should not be part of the TOC. Sorry. for this.

Comment (Firesphere) [reply README.md:99]: No worries :)

**Comment by wernerkrauss** [comment]
Thanks for contributing and fixing the filename issue

**Pull request merged**